### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by alanahodge**
Includes About.md and updated README.md

### CODE REVIEW

**Review by alexlamouri — APPROVED** [review]
Suggestions:
- Add a section on how to expand on the project if someone forks it (new games, >2 players)
- Add deployment instructions for Heroku (info on confluence)
- Mention some challenges/improvements

**Review by NerajM — APPROVED** [review]
👍

**Review by bhawna-sharmaa — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,11 +1,164 @@
...
+5. Deploy your app 
+`$ git push heroku master`
+6. Scale your app
+`$ heroku ps:scale web=1`
+7. Open your app
+`$ heroku open`
+
+## Development
+
+Social Connect is built using `React.js` and `CSS makestyles` styling to create a functional user interface for gameplay.
```
Comment (NavyaGupta) [inline README.md]: not `CSS makestyles`. We used `Material UI` for styling

**Review by NavyaGupta — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alanahodge) [reply README.md]: Oops, typo. Fixed!

**Review by NavyaGupta — APPROVED** [review]
Looks great! Thanks 😍

**Pull request merged**